main/client/sectionHelloFriends/sectionHelloFriends.html

```diff
@@ -0,0 +1,18 @@
+<template name="sectionHelloFriends">
+    <section id="lets-be-friends">
+        <div id="children-pics">
+            <img src="images/gfx_10.jpg">
```

Comment:
Using col-lg-3 adds a padding between each img element which is not like on the mockup.